src/app/(mobile-ui)/card-recovery/page.tsx

```
@@ -0,0 +1,215 @@
+            setStep('done')
+        } catch (e) {
+            setError((e as Error).message || 'Recovery failed — please try again')
+            setStep('preview')
+        }
+    }, [getClientForChain])
+
+    if (!preview && !error) return <PeanutLoading />
+
+    return (
+        <div className="flex min-h-[inherit] flex-col gap-8">
+            <NavHeader title="Recover card funds" onPrev={() => router.push('/home')} />
```

ESLint

[error] Bare router.push/replace as onPrev/onBack creates a parent↔child cycle once the parent uses useSafeBack (the push grows in-app history, useSafeBack pops back to this screen, repeat). Use useSafeBack(parentUrl) — pass { replace: true } to preserve replace semantics. See PR `#1997`.

(no-restricted-syntax)